libminifi/test/TestBase.h

```diff
@@ -246,12 +246,10 @@ class TestPlan {
   void scheduleProcessor(const std::shared_ptr<minifi::core::Processor>& processor);
   void scheduleProcessors();
 
-  // Note: all this verify logic is only used in TensorFlow tests as a replacement for UpdateAttribute
-  // It should probably not be the part of the standard way of running processors
   bool runProcessor(const std::shared_ptr<minifi::core::Processor>& processor, const PreTriggerVerifier& verify = nullptr);
   bool runProcessor(size_t target_location, const PreTriggerVerifier& verify = nullptr);
   bool runNextProcessor(const PreTriggerVerifier& verify = nullptr);
-  bool runCurrentProcessor(const PreTriggerVerifier& verify = nullptr);
+  bool runCurrentProcessor();
   bool runCurrentProcessorUntilFlowfileIsProduced(std::chrono::milliseconds wait_duration);
```

Comment:
Are the other `PreTriggerVerifier` arguments still needed?  The old comment said all of it was for tensorflow only.

Reply:
I tried removing it, but as it turns out the comment was misleading, because the parameter was still used in the `ProcessorTests`, used with a `verifyReporter` lambda, so I only removed the comment.